summoner-cli/src/Summoner/Text.hs

```diff
@@ -26,3 +29,23 @@ headToUpper t = case T.uncons t of
 -- | Convert every element of a list into text, and squash the results
 tconcatMap :: (a -> Text) -> [a] -> Text
 tconcatMap f = T.concat . map f
+
+-- | Aligns a list of texts by their columns
+alignTable :: [(Text, Text, Text)] -> [Text]
+alignTable metas = map (formatTriple maxLengths) metas
+    where 
+      maxLengths :: (Max Int, Max Int, Max Int)
+      maxLengths = getMaxLengths metas 
+
+formatTriple :: (Max Int, Max Int, Max Int) -> (Text, Text, Text) -> Text
+formatTriple (lenA, lenB, lenC) (a, b, c) = 
+    padWord lenA a <> " " <> padWord lenB b <> " " <> padWord lenC c
+
+padWord :: Max Int -> Text -> Text
+padWord maxWordLength word = word <> T.replicate (getMax maxWordLength - T.length word) " "
```

Comment:
Let's rename `padWord` to `padRight` to have a bit clearer name. We actually have this function in another project, so you can just copy-paste the implementation from there:

https://github.com/kowainik/hit-on/blob/caae31f20ee692cbe765bf8f486521267f21ef90/src/Hit/Formatting.hs#L20-L29